__tests__/app.test.js

```diff
@@ -153,16 +153,94 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe("No comments found for article_id: 2")     
         })
     })
-    test("status 400: should return an error messafe when given an article_id that isn't valid", () => {
+    test("status 400: should return an error message when given an article_id that isn't valid", () => {
         return request(app)
         .get('/api/articles/£/comments')
         .expect(400)
         .then(({ body }) => {
-            expect(body.msg).toBe("Invalid ID")
+            expect(body.msg).toBe("Bad request")
         })
     })
   })
-  
+
+
+describe("POST: /api/articles/2/comments", () => {
+    test("status 201: should post a comment for an article and respond with the posted comment", () => {
+        const data = {
+            body: "This is a great article",
+            username: "butter_bridge"
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(data)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.comment.article_id).toEqual(2)
+            expect(body.comment.body).toEqual(data.body)
+            expect(body.comment.author).toEqual(data.username)
+            expect(body.comment).toMatchObject({
+                article_id:expect.any(Number),
+                author:expect.any(String),
+                body:expect.any(String),
+                comment_id:expect.any(Number),
+                created_at:expect.any(String),
+                votes:expect.any(Number)
+            })
```

Comment:
You could just assert that you expect the comment to have the values that you sent and they are on the correct properties - just something to consider 💭